core/src/main/java/org/infinispan/persistence/manager/TransactionalPersistenceManager.java

```diff
@@ -0,0 +1,21 @@
+package org.infinispan.persistence.manager;
+
+import org.infinispan.marshall.core.MarshalledEntry;
+
+import javax.transaction.Transaction;
+import java.util.List;
+
+/**
+ * @author Ryan Emerson
+ */
+public interface TransactionalPersistenceManager extends PersistenceManager {
```

Comment:
@wburns The only problem with separating this into two distinct interfaces is that we rely on the AccessMode enum that is defined in PersitanceManager.java.  I have no objections to simply adding the methods to PersistanceManager, I just thought that the distinction could be beneficial long term, however saying that I can't think of an obvious use-case for when a Tx and non-tx persistence manager would be required